driver-core/src/main/com/mongodb/internal/operation/FindAndUpdateOperation.java

```diff
@@ -70,6 +71,7 @@
     private List<BsonDocument> arrayFilters;
     private Bson hint;
     private String hintString;
+    private BsonValue comment;
```

Comment:
Could this go in BaseFindAndModifyOperation?

Reply:
I followed the existing conventions - like with hint

Reply:
Added JAVA-4544 which I'll do after this ticket :+1:

Reply:
No worries.